### ISSUE TITLE
November 2015 meeting

### ISSUE THREAD

**Issue by waldyrious** [opencompany/www.opencompany.org#168]
I'm starting a new issue (branching off #167) to track the decisions of this month's meeting and what needs to be done. If we agree this is a good model, we can then have one such issue for each monthly meeting. Or perhaps a wiki page would work better so we can all edit in the same place.

Here are the main topics and decisions, mostly actionable items (unchecked items have open questions to resolve or actions to take):

- [x] Complete domain transfer (was already in progress at the time of the meeting, so nothing needed to be done, just wait for the transfer to conclude)
- [ ] Create canonical list of all OCI “holdings” and who handles what:
    - [ ] Question: Where will we put it? On a wiki on this repo? In the README.md?
    - [ ] Question: Is the following list complete? If not, what's missing?
        - [ ] [Website](http://www.opencompany.org/) (hosting @ github, domain w/ @waldyrious)
        - [ ] [Google+ page](https://plus.google.com/+OpencompanyOrg/) and [associated youtube channel](https://www.youtube.com/channel/UCpVVZeu9kQm2Dt8F4oKeMtQ) (currently managed by @galuszkak and @waldyrious). Question: can the channel have a friendlier URL?
        - [ ] Gitter chat rooms [opencompany](https://gitter.im/opencompany) (Question: org. owners only?) and [opencompany/general](https://gitter.im/opencompany/general) (currently managed by 
        - [x] [Twitter profile](https://twitter.com/opencompanyorg) (currently managed by @waldyrious)
        - [x] [Facebook page](https://www.facebook.com/OpenCompanyInitiative) (currently managed by @galuszkak)
        - [ ] [Github organization](https://github.com/opencompany) (Question: Are the members / teams / owners public? Should they be? make 
        - [ ] To do: email newsletter, every 3 or months perhaps, with an online archive to serve as a "blog". We need to figure out how to set this up, technically speaking, and where to gradually collect info to be included in each issue.
- [x] Monthly meetings schedule: On Google Hangouts on Air, on the second sunday of each month, 8pm UTC (subject to adjustments in date or time if necessary)
- [ ] To do: triage current issues and close / resolve as appropriate. We should possibly open a tracking issue (similar to this one) to do that collaboratively.
- [ ] To do: Consolidate teams in the github organization
- [ ] To do: website simplifications (details to be added below)
- [ ] (Many items missing, to be completed from the [document used during the meeting](https://docs.google.com/document/d/16V1Y5zQY1RsIw37YKEghNcKcQaVgQAI2WZiLhGeQ6q4/edit))

**Comment by tenkabuto** [comment]
I updated @waldyrious's post above with all of the items that I found in the document linked to at the bottom and from the notes that I took during the meeting.

I noticed that some of the points that we discussed during the meeting weren't added to the Google Doc, so please look this over, @waldyrious @galuszkak, in case I missed something. :smile:

**Comment by galuszkak** [comment]
Can we move those to seperate tickets? I think having this so long open those help tracking what's in the "month meeting" issue :).

**Comment by waldyrious** [comment]
Agreed, separate issues is the way to go. I'm afraid we might lose some info if we don't confirm whether the notes from the meeting were all added here, but if we block on that we might not move forward at all. (Confirming that all the items were added here shouldn't take too long, though.)
